src/API/Behavior/EmitsEventsTrait.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Behavior;
```

Comment:
I think this would be better placed in the `OpenTelemetry\API\Common\Event`namespace as well.
I know, I introduced these `Behavior` namespaces, but this was before we had the `Common` ones.

Reply:
moved